apiary-extensions-terraform/privileges-grantor/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Terraform module for setting up infrastructure for [Apiary Privilege Grantor](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/apiary-metastore-consumers/privileges-grantor).
+
+For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
+
+## Variables
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| aws\_profile | AWS CLI profile name. | string | n/a | yes |
+| aws\_region | AWS region. | string | n/a | yes |
+| backend\_main\_subnets | Main VPC backend subnets. | list | `<list>` | no |
+| instance\_name | Privilege Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
+| memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
+| metastore\_events\_filter | List of metastore event types to be added to SNS filter. Supported format: `<<EOD "CREATE_TABLE","ALTER_TABLE" EOD` | string | n/a | yes |
+| metastore\_events\_sns\_topic | SNS Topic for Hive Metastore events. | string | n/a | yes |
+| pg\_jars\_s3\_key | S3 key where zip file is located. | string | n/a | yes |
+| pg\_lambda\_bucket | Bucket where the Lambda zip can be found, for example 'bucket_name'. Used together with pg_jars_s3_key. | string | n/a | yes |
+| pg\_lambda\_version | Version of the Privilege Grantor Lambda | string | n/a | yes |
+| pg\_metastore\_uri | Thrift URI of the metastore to which Lambda will connect to. | string | n/a | yes |
+| security\_groups | Security groups in which Lambda will have access to. | list | `<list>` | no |
```

Comment:
not required but no default? what will happen if I leave this empty?

Reply:
same for subnets

Reply:
Not sure how this changed back. It was supposed to be required. I have made it mandatory again.